src/DebugEngineHost/HostMarshal.cs

```diff
@@ -59,6 +59,16 @@ public static IDebugFunctionPosition2 GetDebugFunctionPositionForIntPtr(IntPtr l
         }
 
         /// <summary>
+        /// Obtain a string from a BSTR
```

Comment:
How about:
Obtain the string expression from the bpLocation union for a BPLT_DATA_STRING breakpoint.